hack/cni-plugins/cni-plugins.yaml

```diff
@@ -1,43 +0,0 @@
----
```

Comment:
why are we dropping this yaml?

Reply:
CNI plugins are shipped by CNAO, this is just a remnant of the past.